src/components/AppSidebar/RepeatItem.vue

```diff
@@ -0,0 +1,166 @@
+<!--
+Nextcloud - Tasks
+
+@author Sunik Kupfer
+@copyright 2023 Sunik Kupfer <mail@sunik.de>
+@author Raimund Schlüßler
+@copyright 2018 Raimund Schlüßler <raimund.schluessler@mailbox.org>
+
+This library is free software; you can redistribute it and/or
+modify it under the terms of the GNU AFFERO GENERAL PUBLIC LICENSE
+License as published by the Free Software Foundation; either
+version 3 of the License, or any later version.
+
+This library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU AFFERO GENERAL PUBLIC LICENSE for more details.
+
+You should have received a copy of the GNU Affero General Public
+License along with this library. If not, see <http://www.gnu.org/licenses/>.
+
+-->
+
+<template>
+	<div class="property__item">
+		<div class="repeat__icon">
+			<component :is="icon" :size="20" />
+		</div>
+		<RepeatSummary class="property-repeat__summary__content"
+			:recurrence-rule="recurrence" />
+		<Actions>
+			<ActionButton @click="toggleOptions">
+				<template #icon>
+					<component :is="toggleIcon" :size="20" decorative />
+				</template>
+				{{ toggleTitle }}
+			</ActionButton>
+		</Actions>
+
+		<div v-if="showOptions" class="property-repeat__options">
+			options
+		</div>
+	</div>
+</template>
+
+<script>
+
+import { translate as t } from '@nextcloud/l10n'
+import RepeatSummary from './RepeatItem/RepeatSummary.vue'
+import Pencil from 'vue-material-design-icons/Pencil.vue'
+import Check from 'vue-material-design-icons/Check.vue'
+import { NcActions as Actions, NcActionButton as ActionButton } from '@nextcloud/vue'
+
+export default {
+	components: {
+		RepeatSummary,
+		Actions,
+		ActionButton,
+		Pencil,
+		Check,
+	},
+	props: {
+		recurrence: {
+			type: Object,
+			required: true,
+		},
+		disabled: {
+			type: Boolean,
+			default: false,
+		},
+		placeholder: {
+			type: String,
+			default: '',
+		},
+		icon: {
+			type: String,
+			default: null,
+		},
+	},
+	data() {
+		return {
+			showOptions: false,
+		}
+	},
+	computed: {
+		/**
+		 * The name of the icon for the toggle options button
+		 *
+		 * @return {string}
+		 */
+		toggleIcon() {
+			if (this.showOptions) {
+				return 'Check'
+			}
+			return 'Pencil'
+		},
+		/**
+		 * The text of the toggle options button
+		 *
+		 * @return {string}
+		 */
+		toggleTitle() {
+			if (this.showOptions) {
+				return this.t('calendar', 'Save')
```

Comment:
```suggestion
				return this.t('tasks', 'Save')
```
This needs to be changed everywhere, otherwise the translation will fail.

Reply:
Ah, good to know!